openslides/utils/autoupdate.py

```diff
@@ -16,7 +18,7 @@
 from .collection import Collection, CollectionElement, CollectionElementList
 
 
-def send_or_wait(send_func, *args, **kwargs):
+def send_or_wait(send_func: Any, *args: Any, **kwargs: Any) -> None:
```

Comment:
send_func should always be a channel's send function. Args and kwargs should only be the args and kwargs for such a function.

Reply:
I did not understand all of the functions you write. Please add a commit to the pull request where you add as match typings as possible.

It should be the goal, that there is no Any in the code. (But we will probably not  succeed.)